src/CRCTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\CRC32;
+
+trait CRCTrait
```

Comment:
One thing here: In the file containing the PHP code previously, you indicated you wanted these functions to be private. Putting them in a trait lets classes that need them access them without them being public. I considered putting them in `CRC32`, but then they'd need to remain public. We could put them there and document that they're not intended for use outside the client, or keep them in this trait. WDYT?

Reply:
I prefer the CRCTrait approach.  Does this work all the way back to PHP 5.4? I'm assuming so?

Reply:
Yes, traits were added with PHP 5.4